test/functional/simulator-e2e-specs.js

```diff
@@ -530,6 +530,41 @@ function runTests (deviceType) {
       });
     });
   });
+
+  describe('getUserInstalledBundleIdsByBundleName', function () {
```

Comment:
Does this _need_ a new sim? If not, it would be better to put it in the 'advanced features' section (https://github.com/appium/appium-ios-simulator/blob/master/test/functional/simulator-e2e-specs.js#L336). Otherwise this suite just gets longer and longer (especially on iOS 13, where sim startup is ridiculously slow).

Reply:
Yeah, agree.
I need a simulator created but before launching since we could not find a case, `buildBundlePathMap` raise an error, in current test cases. Let me add the case in existing place.